### ISSUE TITLE
IPv6 inside WireGuard tunnel

### ISSUE THREAD

**Issue by jurajhilje** [ivpn/android-app#29]
## Description

Allow users access to the IPv6 internet after they connect to the WireGuard VPN.

In current clients, WireGuard `[Interface]` is only configured with IPv4 address.
By adding IPv6 address in the client config, VPN tunnel will have IPv6 traffic as well, when connected to gateway which support IPv6.

Example client config:

```
[Interface]
Address = 192.0.2.0,2001:0db8:0:0::2001:0db8/64
ListenPort = 51820
PrivateKey = <PrivateKey>
DNS = 198.51.100.0

[Peer]
PublicKey = <PublicKey>
AllowedIPs = 0.0.0.0/0,::/0
Endpoint = 203.0.113.0:2049
PersistentKeepalive = 25
```

There should be an option in the app settings that enables IPv6 inside VPN tunnel, which is off by default.

**Comment by gorkapernas** [comment]
Verified on version 2.5.0, staging and production.
Devices -> Samsung Galaxy S10 Android 11 and Huawei MediaPad T5 Android 8.
When enabling "IPv6 for WireGuard VPN" , the VPN tunnel will also have IPv6 traffic when connecting to a server, tested with all servers which support IPv6 and I was able to access IPv6 hosts on different browsers.
This only works with WireGuard, once the protocol is changed, only IPv4 traffic will be used.
Also tested WireGuard connectivity (along with different settings, custom DNS, mock location etc) without encountering any obvious issue related to these changes.
